extra/test_release_tool.py

```diff
@@ -419,32 +436,38 @@ def test_get_components_of_type(integration_dir_func):
     # only_non_release=True
     repos_comp = Component.get_components_of_type("git", only_non_release=True)
     repos_name = [r.name for r in repos_comp]
-    assert "deployments" not in repos_name
     assert "deployments-enterprise" not in repos_name
     assert "deviceauth" not in repos_name
     assert "gui" not in repos_name
     assert "integration" not in repos_name
-    assert "inventory" not in repos_name
     assert "inventory-enterprise" not in repos_name
     assert "mender" not in repos_name
     assert "mender-artifact" not in repos_name
     assert "mender-cli" not in repos_name
     assert "tenantadm" not in repos_name
-    assert "useradm" not in repos_name
     assert "useradm-enterprise" not in repos_name
-    assert "workflows" not in repos_name
     assert "workflows-enterprise" not in repos_name
     assert "create-artifact-worker" not in repos_name
     assert "auditlogs" not in repos_name
     assert "mtls-ambassador" not in repos_name
     assert "deviceconnect" not in repos_name
     assert "mender-connect" not in repos_name
     assert "deviceconfig" not in repos_name
+    if not is_staging:
+        assert "deployments" not in repos_name
+        assert "inventory" not in repos_name
+        assert "useradm" not in repos_name
+        assert "workflows" not in repos_name
     # should only include deprecated repos
     assert "deviceadm" in repos_name
     assert "mender-api-gateway-docker" in repos_name
     assert "mender-conductor" in repos_name
     assert "mender-conductor-enterprise" in repos_name
+    if is_staging:
+        assert "deployments" in repos_name
+        assert "inventory" in repos_name
+        assert "useradm" in repos_name
+        assert "workflows" in repos_name
```

Comment:
is this right? I think the location is rather misleading...

Reply:
It is "right" from the `release_tool` point of view: these repos are marked as non-releasable for staging branch (iow we don't tag them when making a HM release)

Reply:
just since it is right underneath the `# should only include deprecated repos` comment - 
```suggestion
    if not is_staging:
        assert "deployments" not in repos_name
        assert "inventory" not in repos_name
        assert "useradm" not in repos_name
        assert "workflows" not in repos_name
    else:
        assert "deployments" in repos_name
        assert "inventory" in repos_name
        assert "useradm" in repos_name
        assert "workflows" in repos_name
    # should only include deprecated repos
    assert "deviceadm" in repos_name
    assert "mender-api-gateway-docker" in repos_name
    assert "mender-conductor" in repos_name
    assert "mender-conductor-enterprise" in repos_name
```

Reply:
Okey, but my point was that deployments, inventory, useradm and workflows are "deprectated" (*) in `staging` branch. I think they should stay together with "deviceadm", "mender-api-gateway-docker", etc, because they are treated the same way.

(*) Instead of "deprecated" I can write "non-releasable repos" or "repos not included in a release", if that will help.

Reply:
but you have the "not include" almost right above in [line 456](https://github.com/mendersoftware/integration/pull/1381/files/d42229c70af12a9c8d153980550a092ebd549f00#diff-a4f8e500e8139d190a4257978c6b52b60639d5423572a8d97fad2d03861bf9ecR456), apart from that I think adding a comment for these would clarify things, since they are very much _not_ deprecated + the entire file looks rather painful to maintain already...

Reply:
Related to @kacf comment above, I will simplify the tests.